python/lsst/ip/isr/isrTask.py

```diff
@@ -1128,16 +1141,30 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         if self.config.doHeaderProvenance:
             # Add calibration provenanace info to header.
             exposureMetadata = inputs['ccdExposure'].getMetadata()
-            for inputName in sorted(inputs.keys()):
+
+            # These inputs change name during this step.  These should
+            # have matching entries in the additionalInputDates dict.
+            additionalInputs = []
```

Comment:
Not that it matters much but couldn't you just set this to `list(inputs.keys())` to start with and append to/extend it?

Reply:
That would certainly work the same, but I think it would require some variable name and comment rewrites.  I think in the long term, this can eventually go away if all cameras switch to `IsrCalib` style brighter-fatter kernels.